docs/getting-started.md

```diff
@@ -55,10 +55,10 @@ npm install dotenv
 
 **Create a .env file**
 
-Create a .env file in the directory to hold your Momento auth token and the TTL (in seconds) you want to use by default.
+Create a .env file in the directory to hold your Momento API token and the TTL (in seconds) you want to use by default.
 
 ```cli
-export MOMENTO_AUTH_TOKEN=<your Momento token here>
+export MOMENTO_API_TOKEN=<your Momento token here>
```

Comment:
man there are a bunch of repos we need to rename this in also. More just thinking outloud. Ill take a pass at other repos in our org.

Reply:
No doubt.